packages/integrations/react/src/index.ts

```diff
@@ -53,6 +53,13 @@ function getRenderer(reactConfig: ReactVersionConfig) {
 	};
 }
 
+export function getContainerRenderer(reactVersion: ReactVersionConfig): ContainerRenderer {
```

Comment:
we don't need to have this passed in. If you look at the code there's a function to inver this already.